### DISCUSSION TITLE
Changing input data

### DISCUSSION THREAD

**Discussion by vartika271987** [magpiemodel/magpie#250] [Q&A] [answered]
Hello team,
I am trying to add an input data file and following all steps in the tutorial, particularly steps 2.2 and 2.3 here - https://github.com/magpiemodel/tutorials/blob/master/6_AdvancedChangeInputs.Rmd

I have done this in the past and have succeeded. However, this time around it's not working and when starting the model it's not able to find the patch file. 
Any ideas what could be wrong?

Thanks!

**Comment by abhimishr** [comment, marked as answer]
Could be an issue with zipping the file or setting the location of this patch.
Can you check if the correct tgz file (your patch) is added to `cfg$input` in `default.cfg` and if the correct location is added to `cfg$repositories`? Also these tgz files have a particular structure - The files is tgz extension but contain a tar file beneath and then the actual patch file should sit inside this tar file. [Line 68 and line 118](https://github.com/magpiemodel/tutorials/blob/master/6_AdvancedChangeInputs.Rmd).

**Comment by vartika271987** [comment]
Thanks Abhi. I realized that I had not added the path to the patched input
data in cfg$repositories. It is fixed and working now.
Thanks!
